test/e2e/network/dns.go

```diff
@@ -19,6 +19,7 @@ package network
 import (
 	"context"
 	"fmt"
+	e2eoutput "k8s.io/kubernetes/test/e2e/framework/pod/output"
```

Comment:
nit: this should be clubbed with other "k8s.io" imports

Reply:
"k8s.io" imports should be a separate block of imports, you can refer to any other file.

Reply:
> "k8s.io" imports should be a separate block of imports, you can refer to any other file.

@yashsingh74  Can you move these two imports to the below section where other k8s.io imports are imported?